class/defaults.yml

```diff
@@ -1,4 +1,177 @@
+                parameters:
+                  compression_mode: none
+            config:
+              metadataPool:
+                replicated:
+                  size: 3
+                  requireSafeReplicaSize: true
+              parameters:
+                compression_mode: none
+              # dataPools rendered from data_pools in Jsonnet
+              preserveFilesystemOnDelete: true
+              metadataServer:
+                activeCount: 1
+                activeStandby: true
+                # metadata server placement done in Jsonnet but can be
+                # extended here
+              mirroring:
+                enabled: false
+            storage_class_config:
+              allowVolumeExpansion: true
+
+
+    node_selector:
+      node-role.kubernetes.io/storage: ''
+
+    tolerations:
+      - key: storagenode
+        operator: Exists
+
+    images:
+      rook:
+        registry: docker.io
+        image: rook/ceph
+        tag: v1.6.6
+      ceph:
+        registry: docker.io
+        image: ceph/ceph
+        tag: v16.2.4
+      cephcsi:
+        registry: quay.io
+        image: cephcsi/cephcsi
+        tag: v3.3.1
+
+    charts:
+      rook-ceph: v1.6.6
+
+    operator_helm_values:
+      image:
+        repository: ${rook_ceph:images:rook:registry}/${rook_ceph:images:rook:image}
+        tag: ${rook_ceph:images:rook:tag}
+      nodeSelector: ${rook_ceph:node_selector}
+      resources:
+        limits:
+          cpu: 1000m
+          memory: 1Gi
+        requests:
+          cpu: 750m
+          memory: 512Mi
+      tolerations: ${rook_ceph:tolerations}
+      csi:
+        enableCSIHostNetwork: false
+        cephcsi:
+          image: ${rook_ceph:images:cephcsi:registry}/${rook_ceph:images:cephcsi:image}:${rook_ceph:images:cephcsi:tag}
+
+    toolbox:
+      enabled: true
+      image: ${rook_ceph:images:rook}
+
+    cephClusterSpec:
+      cephVersion:
+        image: ${rook_ceph:images:ceph:registry}/${rook_ceph:images:ceph:image}:${rook_ceph:images:ceph:tag}
+        allowUnsupported: false
+      dataDirHostPath: /var/lib/rook
+      monitoring:
+        enabled: true
+        rulesNamespace: ${rook_ceph:ceph_cluster:namespace}
+      mon:
+        count: 3
+        allowMultiplePerNode: false
+      network:
+        provider: host
+      placement:
+        all:
+          # nodeAffinity is injected in Jsonnet,
+          # taking placement labels from ${rook_ceph:node_selector}
+          tolerations: ${rook_ceph:tolerations}
+      resources:
+        mgr:
+          limits:
+            cpu: "1"
+            memory: 2Gi
+          requests:
+            cpu: "1"
+            memory: 2Gi
+        mon:
+          limits:
+            cpu: "1"
+            memory: 2Gi
+          requests:
+            cpu: "1"
+            memory: 2Gi
+        osd:
+          limits:
+            cpu: "6"
+            memory: 5Gi
+          requests:
+            cpu: "4"
+            memory: 5Gi
```

Comment:
Is it wise to limit the CPU here? Shouldn't they run on full power when needed?

Reply:
I've never seen the OSDs take more than 4 CPU even under heavy load. Also, since we run both the OSDs and the MONs on the same nodes, we need to make sure the MONs get some CPU even under load. The requests and limits are defined assuming that we have roughly 6-7 vCPUs available per node.

Note that I've used the same limits for all benchmarks until now.